python/fusion_engine_client/messages/control.py

```diff
@@ -163,6 +163,8 @@ class ResetRequest(MessagePayload):
     # @{
     ## Restart the GNSS measurement engine.
     RESTART_GNSS_MEASUREMENT_ENGINE = 0x01000000
+    ## Reboot the FusionEngine processor.
```

Comment:
These should all say "navigation processor" instead of "FusionEngine processor" for consistency with the param name. Same in the C++.